src/renderer/src/components/button/Button.tsx

```diff
@@ -15,6 +16,7 @@ const buttonStyles = cva(
       size: {
         medium: "px-4 py-2",
         large: "px-7 py-2.5",
+        icon: "grid place-items-center aspect-square size-10 p-1",
```

Comment:
Add a negative negative margin to compensate for the white spacing created by the size/padding (`-m-3`), that way the icons are properly aligned with the content.

With `-m-3`
<img width="238" alt="image" src="https://github.com/user-attachments/assets/68c3c47e-008a-4d94-8f45-8d5ea58318ad">

Without it:
<img width="194" alt="image" src="https://github.com/user-attachments/assets/0cc111a9-dcb3-4646-b00f-aef142bc0231">